cci_nepal/analysis/wp2_data_scoping/database_sample/nepal_flood_landslide_descriptive.py

```diff
@@ -0,0 +1,1006 @@
+            ha="center",
+            rotation=20,
+            fontsize=10,
+        )
+
+# %%
+relief_df_gender.plot.barh(width=0.75, figsize=(15, 10), fontsize=16)
+plt.title("Motive for receiving relief by gender", fontsize=20)
+plt.savefig(f"{IMAGE_DIR}/motive_for_receiving_relief_by_gender_side_stacke.png")
+
+# %%
+# why receive relief package by gender
+plt.figure(figsize=(15, 10))
+plt.tight_layout()
+plt.title("Why did you choose to receive relief supplies", fontsize=20)
+data_df["Why did you choose to receive relief supplies"][
+    data_df["Gender of the informant"] == " female"
+].value_counts().plot(kind="barh")
+plt.savefig(
+    f"{project_dir}/outputs/figures/nepal_descriptive/motive_for_receiving_id_male.png"
+)
+
+# %%
+data_df.loc[:, "Why did you choose to receive relief supplies":]
+
+# %%
+cash_spending_df = data_df.loc[
+    :, "For what did you spend the money you received from the Red Cross":
+]
+
+# %%
+# renaming columns
+for col in cash_spending_df.loc[
+    :, "For what did you spend the money you received from the Red Cross":
+].columns[1:10]:
+    cash_spending_df.rename(
+        columns={
+            col: col.replace(
+                "For what did you spend the money you received from the Red Cross", ""
+            ).lstrip()
+        },
+        inplace=True,
+    )
+
+
+# %%
+cash_spending_df.loc[
+    :, "For what did you spend the money you received from the Red Cross":
+].columns[1:11]
+
+# %%
+plt.figure(figsize=(15, 10))
+plt.xlabel("Frequency", fontsize=16)
+plt.ylabel("Spending activity", fontsize=16)
+plt.title("Cash spending", fontsize=20)
+cash_spending_df[
+    cash_spending_df.loc[
+        :, "For what did you spend the money you received from the Red Cross":
+    ].columns[1:10]
+].sum(axis=0).sort_values().plot(kind="barh")
+
+# %%
+# cash_spending_df holds only the data on cash spending. The motive is to allow renaming of fields including others
+# without conflict. - others would appear many times in the main df
+cash_spending_df = cash_spending_df[
+    cash_spending_df.loc[
+        :, "For what did you spend the money you received from the Red Cross":
+    ].columns[1:10]
+]
+
+# %%
+cash_spending_df["Gender of the informant"] = data_df["Gender of the informant"]
+cash_spending_df["age of the informant"] = data_df["age of the informant"]
+
+
+# %%
+female_spending = cash_spending_df[
+    cash_spending_df["Gender of the informant"] == " female"
+]
+male_spending = cash_spending_df[
+    cash_spending_df["Gender of the informant"] == "  male"
+]
+# cash_spending_df[cash_spending_df.columns[:10]][cash_spending_df['Gender of the informant']==' female'].sum(axis=0).sort_values().plot(kind='barh')
+
+# %%
+female_spending[female_spending.columns[:9]].sum(axis=0).sort_values()
+
+# %%
+male_spending[male_spending.columns[:9]].sum(axis=0).sort_values()
+
+# %%
+cash_spent_by_gender_labels = [
+    "paid the loan taken from neighbourrelatives",
+    "spent unnecessarily",
```

Comment:
There seems to be a few cases where you have written out the values and their counts as lists in the script? Is this because you are re-naming them? I would try to automate this process as a function if possible, as it seems to be repeated a few times?

Reply:
If its not to do with re-nenaming, you can get a list of unique values in a column using the below:

df['col'].unique().tolist()